### DISCUSSION TITLE
The future of this library

### DISCUSSION THREAD

**Discussion by tealeg** [tealeg/xlsx#663] [General]
Hi everyone,

I want to talk openly about the future of this library.   I've been working on this behemoth for over a decade now, and it's been nearly that long since I've actually used it myself.   Right now, as at several times in the past, there is a growing queue of work to be done, and I have nearly zero time to look at it.   In the past, I have been lucky to find people who were actively able to help with the maintenance of this library and I am extremely grateful for that, but for now, it seems I am alone. 

I'm open to any creative suggestions at this point.

-- tealeg

**Comment by vladimirfilin** [comment]
This behemoth is very useful! I checked out a few other libraries to work with xlsx from Go. This code works better than others. Therefore, I am in favor of maintaining this library as much as possible for the author. I tried to fix a couple of bugs myself, but it seems that my knowledge of Go is not up to the required level.

**Reply by tealeg** [reply]
@vladimirfilin I'm very happy to give tips and of course review any PRs.  I can't really commit to always respond quickly, but if you can live with me being slow feel free to ask questions.

**Comment by Siva-Karthi** [comment]
I'm Interested @tealeg

**Reply by tealeg** [reply]
@Siva-Karthi OK.  What are you interested in doing?

**Reply by Siva-Karthi** [reply]
@tealeg I'm interested in coding contribution in this repo. If you wanted me to work on adding new features and bug fixes, you can approach me

**Reply by tealeg** [reply]
@Siva-Karthi you are more than welcome to work on any issue.  We currently have two that are marked as "low-hanging fruit" which would be relatively gentle starting points.   See: https://github.com/tealeg/xlsx/issues?q=is%3Aissue+is%3Aopen+label%3Alow-hanging-fruit

The highest value issues are the ones marked "Critical", but they often require a fairly full understanding of the library. 

  If you'd need pointers or guidance let me know.

**Comment by dolmen** [comment]
If this project is not actively maintained, It might be helpful for potential users to add to the README a section pointing to alternative packages and reviews of the "XLSX in Go" ecosystem.

In my case I have used [excelize](https://pkg.go.dev/github.com/360EntSecGroup-Skylar/excelize/v2) on some projects, and I wonder if it's time to migrate my xlsx v1-based projects to it.
